pom.xml

```diff
@@ -236,8 +237,7 @@
                         <configuration>
                             <referenceVersion>5.24ea0</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <!-- Reduce the API surface of ChronicleReader -->
-                            <binaryCompatibilityPercentageRequired>100</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>99.9</binaryCompatibilityPercentageRequired>
```

Comment:
The return type of `RollCycles.all()` changed from `Iterable<RollCycles>` to `Iterable<RollCycle>`